types/v56/openapi.go

```diff
@@ -563,6 +563,17 @@ type DefinedEntity struct {
 	State      *string                `json:"state,omitempty"`      // Every entity is created in the "PRE_CREATED" state. Once an entity is ready to be validated against its schema, it will transition in another state - RESOLVED, if the entity is valid according to the schema, or RESOLUTION_ERROR otherwise. If an entity in an "RESOLUTION_ERROR" state is updated, it will transition to the inital "PRE_CREATED" state without performing any validation. If its in the "RESOLVED" state, then it will be validated against the entity type schema and throw an exception if its invalid
 	Owner      *OpenApiReference      `json:"owner,omitempty"`      // The owner of the defined entity
 	Org        *OpenApiReference      `json:"org,omitempty"`        // The organization of the defined entity.
+	Message    string                 `json:"message,omitempty"`    // A message field that might be populated in case entity Resolution fails
```

Comment:
This is curious, as I didn't find it in the [docs](https://developer.broadcom.com/xapis/vmware-cloud-director-openapi/latest/data-structures/DefinedEntity/). I had the doubt whether this was new or not, but can't check I guess.

Reply:
I found this being present mainly when resolving an RDE fails. In such cases it puts in a message.